includes/class-wc-everypay-renderer.php

```diff
@@ -18,23 +18,34 @@ public function __construct($helpers, $public_key, $tokenization_status)
 
 	public function render_iframe($amount, $max_installments)
 	{
-		global  $current_user;
-		$billing_address = get_user_meta( $current_user->ID, 'billing_address_1', true );
-		$total = $this->helpers->format_amount($amount);
+		// global $current_user;
+		// $billing_address = get_user_meta($current_user->ID, 'billing_address_1', true );
+		// $billing_email = get_user_meta($current_user->ID, 'billing_email', true);
+		// $billing_phone = get_user_meta($current_user->ID, 'billing_phone', true);
+
+        // @note support for guest checkout
+    	$billing_address = WC()->customer->get_billing_address();
+    	$billing_email = WC()->customer->get_billing_email();
+    	$billing_phone = WC()->customer->get_billing_phone();
+
+        $total = $this->helpers->format_amount($amount);
```

Comment:
Super minor: While we should use spaces to be PSR compliant, I agree to have tabs in these files for consistency. 

However before $total we have spaces. Also I think we need 2 tabs instead of 1 in the above lines.

Reply:
updated